doc/index.md

```diff
@@ -23,117 +25,6 @@ Static analysis     | GoReport              | [![Go Report Card](https://gorepor
 Translations        | Weblate               | [![Translation status](https://hosted.weblate.org/widgets/linux-containers/-/svg-badge.svg)](https://hosted.weblate.org/projects/linux-containers/lxd/)
 Project status      | CII Best Practices    | [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1086/badge)](https://bestpractices.coreinfrastructure.org/projects/1086)
 
-## Installing LXD from packages
```

Comment:
Moved this content to a separate page - will need to add a redirect (how is this currently done?).

Reply:
One thing to be careful about when messing with `index.md` is that this is also the README showed at https://github.com/lxc/lxd so we need to make sure that this one renders perfectly on Github and that it still has everything that someone needs to understand what LXD is at a glance and how to make use of the repo they're looking at.

If this becomes too problematic, we could revert to having a completely separate README.md for that purpose.

Reply:
Which is a good argument for keeping this file small and concise. Right now, it's a collection of information that doesn't fit anywhere else.
Imo, this file should contain the logo, the general intro to what LXD is, project information (status, bug reporting/contributing/support) and links to where to find more information. Installation should be moved out to a separate page (and linked, of course). And having an FAQ in two different places is extremely confusing.

But sure, we can take that in a separate PR.